native/src/response.js

```diff
@@ -0,0 +1,15 @@
+export function error({ message }) {
+  return responsify({ "status": "error", "errors": [message]});
+}
+
+export function fatal({ message }) {
+  return responsify({ "status": "fatal", "errors": [message]});
+}
+
+export function ok(ast) {
+  return responsify({ast, "status": "ok"});
+}
+
+function responsify(resp) {
+  return resp;
```

Comment:
Oh, yes. This was funny. Original implementation was something like:

```js
return { ...defaults, ...resp };
```

With defaults being the driver, language and language version but then they got removed. Let me check with @smola if that is expected to change.